src/Mapscii.js

```diff
@@ -101,8 +97,8 @@ class Mapscii {
       this.width = config.size.width;
       this.height = config.size.height;
     } else {
-      this.width = config.output.columns >> 1 << 2;
-      this.height = config.output.rows * 4 - 4;
+      this.width = config.width ? config.width : config.output.columns >> 1 << 2;
```

Comment:
Why introduce `config.width` and `config.height` if there's already `config.size` with both these values?

Reply:
This was intended as a preliminary way to expose some options via command line parameters, including width and height. As they are nested via the size structure, a (simple) parser would be required to set these. I don't have any experience with node, so I don't know how/where to handle parsing of command line arguments in this case.

Reply:
All args will need to be validated anyway. Those numbers will need to be parsed to the number type as well.
As part of these validations and transformations we can have them override the values from the config.

Reply:
Setting width and height manually will interfere with autoresizing - implying: if width/height is given as argv, autoresize should be disabled.

Reply:
Does https://github.com/rastapasta/mapscii/blob/393f867c847f5789947347f0016df463aae3cfcf/src/Mapscii.js#L86-L106 not effectively disable autoresizing?